pkg/bindings/manifests/manifests.go

```diff
@@ -227,6 +227,10 @@ func Push(ctx context.Context, name, destination string, options *images.PushOpt
 	if options.SkipTLSVerify != nil {
 		params.Set("tlsVerify", strconv.FormatBool(!options.GetSkipTLSVerify()))
 	}
+	// Add compressions if any
+	for _, compression := range options.AddCompression {
+		params.Add("addCompression", compression)
+	}
```

Comment:
`options.ToParams` doesn’t handle this? As the comment above says, `SkipTLSVerify` is special, and all of the other options in `options` are not added individually in this function.

(Note that I didn’t try this in practice, and I didn’t even read `ToParams` carefully, so I don’t know what I don’t know.)

Reply:
Corrent `ToParams` handles it internally as well, removed this.